src/CommunityToolkit/Xamarin.CommunityToolkit/Effects/Touch/PlatformTouchEffect.ios.cs

```diff
@@ -13,14 +13,16 @@ namespace Xamarin.CommunityToolkit.iOS.Effects
 {
 	public class PlatformTouchEffect : PlatformEffect
 	{
-		TouchUITapGestureRecognizer touchGesture;
+		UIGestureRecognizer touchGesture;
 
-		UIHoverGestureRecognizer hoverGesture;
+		UIGestureRecognizer hoverGesture;
 
 		TouchEffect effect;
 
 		UIView View => Container ?? Control;
 
+		bool IsiOS13OrNewer => UIDevice.CurrentDevice.CheckSystemVersion(13, 0);
```

Comment:
Should we move this to somewhere where everyone can use this? We probably need this more often?

I know Forms has these checks too, but I guess they're internal? Or do they have something for this?

Reply:
Agreed. I will move it to some helper class

```var x = XCT.IsiOS13OrNewer;```

Does it look good?

Reply:
Done